src/size-rhythms.md

```diff
@@ -0,0 +1,65 @@
+# Size rhythms
+
+When designing components and layouts, you will have to specify size of many properties such as `padding`, `margin`, `grid-gap`, `width`, `height`, and so forth.
+
+To generate a consistent layout and visual expression, we only use sizes that are multiples of four (4 * x). For example 4, 8, 12, 16, 20, and so on.
+
+It is very important to follow this convention, both when creating new components for Lime Elements, and when creating bigger components and layouts which use these components. When the entire interface is calculated based on the single core value of 4, elements of the UI will align auto-magically.
+
+For instance, if you want to create a custom row of buttons (not directly using Lime Elements) with this sizing convention, its `height` and `line-height` can be 40 units (to vertically align the text inside the button), the `padding-left` and `padding-right` can be 12 units, and its `margin-left` and `margin-right` which set the distance to the next or previous buttons can be for example 4 or 2, or in this example 6.
+
+Even though 6 is not a multiple of 4, it sums up to a 12 units distance between two buttons in a row; and since 12 is a multiple of 4, the result will be moa👌.
```

Comment:
What does "moa" mean?

Reply:
it's a "sound" you make when something is perfect and you appreciate it 😄  while you do this gesture -> 👌 with you hand in front of your mouth. Like:
😚👌 moa....

But let's change it to. 😂

Reply:
Yeah, I think it's better to change it. We don't need to be stick-up-the-butt formal in our communication, but for one-way communication like this (we're it's not necessarily easy or even possible for the reader to ask for clarification), I think we should prioritise clarity over being casual and personal 🙂